pkg/apis/core/validation/validation_test.go

```diff
@@ -6819,10 +6830,20 @@ func TestValidateResizePolicy(t *testing.T) {
 			},
 			true,
 			field.ErrorList{field.Duplicate(field.NewPath("field").Index(2), core.ResourceCPU)},
+			"Always",
+		},
+		"InvalidPolicyWithPodRestartPolicy": {
```

Comment:
Just for completeness, could you please test cases for:
- cpu: RestartContainer, memory: NotRequired
- cpu: RestartContainer, memory: RestartContainer
- cpu: NotRequired, memory: NotRequired

Reply:
Done